app/src/main/java/com/strv/movies/ui/navigation/MoviesDestinations.kt

```diff
@@ -0,0 +1,7 @@
+package com.strv.movies.ui.navigation
+
+object MoviesDestinations {
+    const val MOVIES_LIST_ROUTE = "movies_list"
+    const val MOVIE_DETAIL_ROUTE = "movie_detail"
+    const val MOVIE_ID_KEY = "movie_id"
```

Comment:
do we wanna combine destinations and ids? I would split it to avoid confusion

Reply:
Good point. I was thinking about that too when I started to work on ViewModels. I will split it into two objects.